tests/memcached/memcached.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+# Copyright (C) 2018 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+load ../utils
+
+
+@test "Start memcached server" {
+    sudo docker run --name memcached-server --rm -d clearlinux/memcached
```

Comment:
In the CI, could we have the user in the docker group so sudo isn't needed for these tests?